### PR TITLE
Debugged and improved surface2ind_topo with interpolation options

### PR DESCRIPTION
**Pull request by thast**
Debugged and improved surface2ind_topo with interpolation options

- Works on both 3D and 2D tensor mesh.
- User can now choose the interpolation method ('nearest', 'linear', 'cubic')

### CODE REVIEW

**Comment by thast** [comment]
job 3541 18 - simpeg simpeg - travis ci 2017-05-12 15-58-20

**Comment by thast** [comment]
Here is the output of the example
figure_1

**Comment by rowanc1** [comment]
This is beautiful. Thanks @thast!

File: SimPEG/Utils/modelutils.py [hunk 1]
```diff
@@ -1,62 +1,61 @@
...
 import numpy as np
 
-def surface2ind_topo(mesh, topo, gridLoc='CC'):
-# def genActiveindfromTopo(mesh, topo):
+def surface2ind_topo(mesh, topo, gridLoc='CC', method='cubic',fill_value=np.nan):
     """
     Get active indices from topography
     """
-
+    from scipy.interpolate import griddata
```
Comment (rowanc1) [inline SimPEG/Utils/modelutils.py]: Imports should be done at the top of the file ideally (inside functions is a performance hit...).

**Comment by thast** [comment]
I feel this is ready for merging. @rowanc1 and @lheagy for your feedbacks.

**Pull request merged**